### ISSUE TITLE
Temporary sudden big increase of highlight number on the float - likely a Firefox defect

### ISSUE THREAD

**Issue by peter-kehl** [ldenoue/yawas#29]
Most likely a Firefox defect. So far this defect has been seen only once. For now this issue is to track any occurrences/circumstances.

I was adding standard highlights, on a pretty much static web page: https://mozilla.github.io/policy-templates/. As I'm adding highlights happily and I don't track the number of the highlights shown by the Yawas float. But suddenly I notice a too high number: it jumps from a small value (a little above 10 or 20) to 435 (side note: ignore the white-on-black inversion):
![image](https://github.com/ldenoue/yawas/assets/4270240/a2e84dbb-b868-4945-b03f-67201ea52f50)

I am not sure if I added one more highlight before I copied the existing stored title (in the following step), as I was in a flow, and I may have noticed the `435` number as I was already clicking Yawas to add the next highlight.

I click the toolbar "Edit this bookmark (Ctrl+D)" button, I copy the title (with the highlights):

`policy-templates | Policy Templates for Firefox#__#<<policies.json>> <<Linux>> <<file goes into firefox/distribution>> <<where firefox is the installation directory for firefox>> <<or you can specify system-wide>> <<file in /etc/firefox/policies>> <<domains allowed to access Google Workspace>> <<Block access to the Add-ons Manager (about:addons)#red>> <<Block access to about:config.#red>> <<Block access to About Profiles#red>> <<about:profiles#green>> <<default download directory>> <<Turn off application updates>> <<Disable the built in PDF viewer>> <<Disable Firefox Accounts integration (Sync).>> <<Remove access to Firefox Screenshots.>> <<Disable the “Forget” button.>> <<Disable the Refresh Firefox button#red>> <<Prevent system add-ons#red>> <<Don’t check if Firefox is the default browser at startup.#red>> <<Control the installation, uninstallation and locking of extensions.>> <<a list of bookmarks managed#green>> <<that cannot be changed by the user#red>> <<Allow manual updates only#green>> <<do not notify the user about updates.#red>> `

I count the number of `<<`  and `>>`, there are 25 of each, all paired, and seem well-formed.

Now, I must have added two highlights, before I realized I could refresh the page in the next step - I don't remember the exact steps. But, from memory, the number of the highlights stayed incorrect (`4xx`) and I don't remember if it got increased by 1-2 or not - but either way, the `4xx` number didn't get fixed on its own, until I refreshed the page:

I refresh the page, and now the Yawas float shows a correct number: 1/27. And the stored title got two more highlights added:

`policy-templates | Policy Templates for Firefox#__#<<policies.json>> <<Linux>> <<file goes into firefox/distribution>> <<where firefox is the installation directory for firefox>> <<or you can specify system-wide>> <<file in /etc/firefox/policies>> <<domains allowed to access Google Workspace>> <<Block access to the Add-ons Manager (about:addons)#red>> <<Block access to about:config.#red>> <<Block access to About Profiles#red>> <<about:profiles#green>> <<default download directory>> <<Turn off application updates>> <<Disable the built in PDF viewer>> <<Disable Firefox Accounts integration (Sync).>> <<Remove access to Firefox Screenshots.>> <<Disable the “Forget” button.>> <<Disable the Refresh Firefox button#red>> <<Prevent system add-ons#red>> <<Don’t check if Firefox is the default browser at startup.#red>> <<Control the installation, uninstallation and locking of extensions.>> <<a list of bookmarks managed#green>> <<that cannot be changed by the user#red>> <<Allow manual updates only#green>> <<do not notify the user about updates.#red>> <<Disable or configure PDF.js>> <<primary (formerly master) password>> `

![image](https://github.com/ldenoue/yawas/assets/4270240/941ac757-1b3c-497a-8906-695041bd496a)

Firefox 122.0 686 (32-bit) binary downloaded from https://ftp.mozilla.org/pub/firefox/releases/122.0/linux-i686/en-US, on current Manjaro x64 X11 GNOME Linux. I'll try 122.0.1 soon.

**Comment by peter-kehl** [comment]
tl;dr: No pattern in sudden number jumps. Some errors in the Browser Console.

After a Firefox restart (still the same version of Firefox 122.0 x32) after adding a few highlights the float now shows `1/274 highlights`.

Surprisingly, the Yawas button shows `0`. (Side notes: I didn't have the Yawas button in the toolbar in the previous screenshots - too crowded toolbar. I do push the envelope with my Firefox too much. If this occurs with other Firefox versions, I'll bisect disable/enable other extensions - though at the moment I don't have many, and most of them are not bookmark-related.)
![image](https://github.com/ldenoue/yawas/assets/4270240/1a48ffe4-4b8f-4fe6-9a22-fc41512275b1)


Unsure if it's caused/related to errors in Browser Console:
![image](https://github.com/ldenoue/yawas/assets/4270240/a90ea0da-65f5-4468-a9c8-81ea6f3be49f)


Side note: The same Browser Console, but for "Parent process only (Fast)" is showing "62 hidden" matches for `yawas`, but I wasn't able to actually show/unhide them:
![image](https://github.com/ldenoue/yawas/assets/4270240/bf63f028-3848-48a5-9508-7116bfc95f35)

**Comment by peter-kehl** [comment]
In addition to the above errors, one more type of repeated errors in Browser Console (and, again, unsure if it's related to this bug):
![image](https://github.com/ldenoue/yawas/assets/4270240/3cba661c-e2e7-466e-a150-87ee748139d5)
```Promise rejected after context unloaded: Actor 'Conduits' destroyed before query 'RuntimeMessage' was resolved```
